client/lib/desktop/index.js

```diff
@@ -152,7 +152,7 @@ const Desktop = {
 				if ( loaded ) {
 					debug( 'Gutenberg iframe loaded' );
 
-					const evt = new Event( 'editor-loaded' );
+					const evt = new Event( 'editor-loaded' ); /* eslint-disable-line no-undef */
```

Comment:
Same here: `window.Event`, or, if the code runs in the Node.js server context, too, `globalThis.Event`.

Reply:
Addressed in f631044.

The preload script consuming this event in Desktop has node integration disabled, so we can expect this code to run only in the renderer context.